src/publicReportingAPI.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import ClientPublicReportingAPI from '@reportportal/client-javascript/lib/publicReportingAPI';
+import { RP_STATUSES } from '@reportportal/client-javascript/lib/constants/statuses';
+
+export const PublicReportingAPI = {
+  setLaunchStatusPassed: (): void => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.PASSED),
```

Comment:
It is also required to provide reporting methods without explicit data (as [here](https://github.com/reportportal/agent-js-mocha/blob/master/lib/publicReportingAPI.js#L41)).
This can be useful when the user doesn't know in advance what type of `status` will be in the code when using a variable for it.